osg-token-renewer.py

```diff
@@ -38,46 +41,63 @@ def validate_config_dict(cfgx):
     for token in cfgx["token"]:
         account = cfgx["token"][token].get("account")
         if not account:
-            print("token %s: missing 'account' attribute" % token,
-                  file=sys.stderr)
-            return False
+            return emsg(f"token {token}: missing 'account' attribute")
         elif account not in cfgx["account"]:
-            print("token %s: missing 'account %s' section" % (token, account),
-                  file=sys.stderr)
-            return False
+            return emsg(f"token {token}: missing 'account {account}' section")
         elif not cfgx["account"][account].get("password_file"):
-            print("account %s: missing 'password_file' attribute" % account,
-                  file=sys.stderr)
-            return False
+            return emsg(f"account {account}: missing 'password_file' attribute")
+        elif not cfgx["token"][token].get("token_path"):
+            return emsg(f"token {token}: missing 'token_path' attribute")
```

Comment:
Can you clarify these error messages so that it's clear that these are issues with the config file?

Reply:
Sure thing; updated.